### PR TITLE
Preprod: shortstory: The Man who Saw the Future (1930) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
Rating 3/5.

### CODE REVIEW

File: notes/stories/literature/shortstory-amazingstories.st.txt [hunk 1]
```diff
@@ -1221,3 +1221,40 @@ folk belief [Some of the workmen thought that an Incan devil or evil spirit had
...
+
+:: Ratings
+3 <paul>
+
+:: Choice Themes
+what if I found myself in the future [The story follows the apothecary's assistant Henri Lothiere as he is transported by a pair of scientists from 1444 Paris to the Paris of 1944.]
+
+:: Major Themes
+time travel [The two scientists Rastin and Thicourt used some or another machine they'd constructed to transport Henri Lothiere from 1444 Paris to their own time and place of 1944 Paris.]
+what if I told the truth and nobody would believe me [Nobody in Henri Lothiere's own time of 1444 believed his story that he'd traveled to 1944 Paris and see numerous technological marvels. Instead, he was labeled a sorcerer and a blasphemer and ultimately burned at the stake.]
```
Comment (odinlake) [inline notes/stories/literature/shortstory-amazingstories.st.txt:1251]: Brings also to mind
https://www.themeontology.org/theme.php?name=science%20as%20magic%20to%20the%20primitive

**Review by odinlake — APPROVED** [review]
Suggestions inline

File: notes/stories/literature/shortstory-amazingstories.st.txt [hunk 2]
```diff
@@ -1221,3 +1221,40 @@ folk belief [Some of the workmen thought that an Incan devil or evil spirit had
...
+=============================================
+
+:: Title
+The Man who Saw the Future
+
+:: Date
+1929-12
+
+:: Description
+A 15th century Frenchman is plucked out of his own time and given a tour of
```
[the remark this answers is not in the document]
Comment (paul-sheridan) [reply notes/stories/literature/shortstory-amazingstories.st.txt:1234]: Good call. I always seem to neglect this one, somehow.

File: notes/stories/literature/shortstory-amazingstories.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/literature/shortstory-amazingstories.st.txt:1251]: I deliberately left this out at first, but I found some content from one paragraph to justify its application. Added accordingly!

**Pull request merged**